app/assets/stylesheets/layout/_work_packages_full_view.sass

```diff
@@ -140,6 +142,15 @@ body.controller-work_packages.full-create
     display: block !important
     visibility: hidden
 
+.work-packages-full-view--resizer
+  .work-packages--resizer
+    height: 100vh
+    width: 18px
+    left: -2px
+    &::before
+      position: sticky
```

Comment:
At least for me it does not do the trick. At first the handles stays at a fixed position but if I scroll further down, the handle disappears.

Reply:
Same here

Reply:
Yes, your are right. Thanks! I fixed this. Hope this solution is working for you as well.